.eslintrc.js

```diff
@@ -1,15 +1,25 @@
 module.exports = {
   root: true,
-  extends: '@react-native-community',
+  extends: ['@react-native-community', 'plugin:@typescript-eslint/recommended-requiring-type-checking'],
```

Comment:
Update config same as project-typescript for consistency

Reply:
Updated!